client/post-editor/edit-post-status/index.jsx

```diff
@@ -1,48 +1,53 @@
 /**
  * External dependencies
  */
-const React = require( 'react' ),
-	noop = require( 'lodash/noop' );
+import React, { PropTypes, Component } from 'react';
+import noop from 'lodash/noop';
```

Comment:
Legacy, but while you're here would encourage importing from root Lodash package instead (recommended since #6539).

```js
import { noop } from 'lodash';
```